src/locales/ja-JP.ts

```
@@ -0,0 +1,144 @@
+    completeText: '更新成功',
+  },
+  tour: {
+    prevStepText: '前へ',
+    completeText: '完了',
+    nextStepText: '次へ',
+  },
+  watermark: {
+    errorCanvasTips: '現在の環境はCanvasをサポートしていません',
+  },
+}
+export default jaJP
```

ESLint

[error] Insert `⏎`

(prettier/prettier)